zerberus/zerberusgui.h

```diff
@@ -60,13 +60,17 @@ class ZerberusGui : public Ms::SynthesizerGui, Ui::ZerberusGui {
       QString _loadedSfName;
       QProgressDialog* _progressDialog;
       QTimer * _progressTimer;
-      std::vector<struct SfzNamePath> _sfzToLoad;
+      std::list<struct SfNamePath> _sfzToLoad;
```

Comment:
Not sure here whether `std::vector` isn't preferred over `std::list`

Reply:
I used std::list because I needed to pop_front. In master version the list of sound fonts is completed in reverse alphabetical order. To fix it I used pop_front instead pop_back.